### PR TITLE
WiP ⛔   feat: hello world

### PR DESCRIPTION
**Pull request by amar771**
* added main.py
* main.py prints "hello world test"

Refs: [RAS-137](https://hycu.atlassian.net/browse/RAS-137)

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1 +1 @@
-print("not hello world")
+print("Hello world test")
```
Comment (amar771) [inline main.py]: should be just `Hello World` without `test`

Comment (amar771) [reply main.py]: please change

**Review by amar771 — COMMENTED** [review]
Everything is okay, except this line, please change 🙏

File: main.py (hunk 1 above)
Comment (amar771) [reply main.py]: changed, will resolve

**Review by amar771 — COMMENTED** [review]
LGTM 🚀

**Review by ShifterIWNL — APPROVED** [review]
Looks good to me [raketica]

**Pull request merged**